mls-rs/Cargo.toml

```diff
@@ -16,7 +16,7 @@ features = ["external_client", "sqlite"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-default = ["std", "rayon", "rfc_compliant", "tree_index", "fast_serialize"]
+default = ["std", "rayon", "rfc_compliant", "tree_index", "fast_serialize", "last_resort_key_package_ext"]
```

Comment:
By the way, there is no particular need for this to be a default feature — I just figured that others would like to use it too. But let me take it out for simplicity.